content/flux/v0.x/data-types/composite/dict.md

```diff
@@ -3,11 +3,120 @@ title: Work with dictionaries
 seotitle: Work with dictionaries in Flux
 list_title: Dictionary
 description: >
-  ...
+  Learn how to work with dictionaries in Flux.
 menu:
   flux_0_x:
     name: Dictionary
     parent: Composite types
 weight: 203
 flux/v0.x/tags: ["composite types", "data types"]
----
\ No newline at end of file
+related:
+  - /flux/v0.x/stdlib/dict/
+---
+
+A **dictionary** type is a collection of key-value pairs
+in which keys can be any type, but must all be the same type.
+Values can also by any type, but must all be the same type.
+
+###### On this page:
+- [Dictionary syntax](#dictionary-syntax)
+- [Reference dictionary values](#reference-dictionary-values)
+- [Operate on dictionaries](#operate-on-dictionaries)
+
+## Dictionary syntax
+A **dictionary** literal contains a set of key-value pairs (also known as elements)
```

Comment:
are key-value pairs only referred to as elements in the context of a dictionary?

Reply:
Right now, yes, but this may change. Dicts are young enough that terminology conventions haven't fully settled.